WBooks/Localizable/es.lproj/Localizable.strings

```diff
@@ -9,19 +9,33 @@
 /* NAVIGATION CONTROLLER TITLES*/
 "TITLE_LIBRARY" = "LIBRERIA";
 "TITLE_WISHLIST" = "LISTA DE DESEO";
-"TITLE_ADDNEW" = "AGREGAR NUEVO";
+"TITLE_ADDNEW" = "AGREGAR LIBRO NUEVO";
 "TITLE_RENTALS" = "ALQUILER";
 "TITLE_SETTINGS" = "CONFIGURACIÓN";
 "TITLE_DETAILBOOK" = "DETALLE";
 
 /* TITLES BUTTONS */
 "BUTTON_WIHSLIST" = "AGREGAR A LA LISTA DE DESEOS";
 "BUTTON_RENT" = "RENTAR";
+"BUTTON_SUBMIT" = "ENVIAR";
 
 /* ALERTS */
 "TITLE_INFORMATION" = "INFORMACIÓN";
 "TITLE_ERROR" = "ERROR";
 "BUTTON_DONE" = "ACEPTAR";
 "ERROR_UNAVAILABLE" = "El libro no se encuentra disponible";
 "ERROR_GENERAL" = "No se ha podido rentar el libro, intentelo mas tarde";
+"ERROR_ADDBOOK" = "No se ha podido añadir el libro, intentelo mas tarde";
```

Comment:
ERROR_GENERAL == ERROR_ADDBOOK

Reply:
They are very similar, just change one word, maybe you should change the name from ERROR_GENERAL to ERROR_RENTAL